workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/CommandDiffDeserializerGenerator.tt

```diff
@@ -0,0 +1,75 @@
+<#@ template language="C#" #>
+<#@ output extension=".cs" #>
+<#
+    var fieldDetailsList = GetFieldDetailsList();
+    var componentDetails = GetComponentDetails();
+    var generatedHeader = CommonGeneratorUtils.GetGeneratedHeader();
+    var commandDetailsList = GetCommandDetailsList();
+    var eventDetailsList = GetEventDetailsList();
+    var componentNamespace = qualifiedNamespace + "." + componentDetails.ComponentName;
+    var profilingStart = $"Profiler.BeginSample(\"{componentDetails.ComponentName}\");";
```

Comment:
unused profiling markers, please use them were useful or remove these

Reply:
There is a lot of unneeded stuff in the codegen. My plan was to sort that out after they've settled slightly.

Reply:
Screw it I cleaned that up now.